modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -150,13 +150,21 @@ A `Time` in the current `TimeZone`.
 [[dw_type_dates_period]]
 === Period
 
-Periods have the form:
+A `Period` represents an amount of time. The type takes the following form:
 
 - `P[n]Y[n]M[n]DT[n]H[n]M[n]S`
 - `P<date>T<time>``
 
 Where the [n] is replaced by the value for each of the date and time elements that follow the [n].
 
+`Period` has two subcategories:
+
+* `DatePeriod` +
+ A date-based amount of time, for example, 1 year, 2 months, 3 days. Because these expressions are date-based, they cannot be converted to milliseconds. For example, a month does not specify whether it is a month of 31 days or not, nor does a year specify whether it is a leap year. The data expresses just an amount of date. In DataWeave, the syntax is `|P1Y|`.
```

Comment:
And...should we use "convert" here when later text uses "coerce"?

I know you do not have time for it now, but I wonder if we want to reorganize this whole set of topics into "Date-Based and Time-Based Periods, and then move around texts so that there is one subtopic for each?